schema/User.re

```diff
@@ -13,30 +15,37 @@ type t = {
 
 type user = t;
 
-module ModelConfig = {
-  type t = user;
-  let table = "app_users";
-  let parseRow = row => {
-    id: row[0],
-    email: row[1],
-    role: row[2] == "ADMIN" ? Admin : User,
+let parseRole = roleString =>
+  switch (roleString) {
```

Comment:
why role is `option(string)`?

Reply:
something wrong on the initial schema, gonna fix on another pr, this is not the only one wrong